src/VoiceConnection.ts

```diff
@@ -192,6 +194,12 @@ export class VoiceConnection extends TypedEmitter<VoiceConnectionEvents> {
 		state: GatewayVoiceStateUpdateDispatchData | undefined;
 	};
 
+	/**
+	 * The receiver of this voice connection. You should join the voice channel with `selfDeaf` set
+	 * to false for this feature to work properly.
```

Comment:
I was told it didn't matter, and that bots received voice packets regardless of whether they were deaf or not, is this true?

Reply:
This is true for self mute - bots that are muted can still speak. However, bots that are deafened don't receive audio.